app/models/role.rb

```diff
@@ -1,6 +1,8 @@
 # @abstract
 class Role < ApplicationRecord
   include HasContentId
+  include PublishesToPublishingApi
```

Comment:
There's a need to pass the correct RolePresenter for this to work properly

Reply:
(this has been addressed here: https://github.com/alphagov/whitehall/pull/4032/files#diff-b11bd5eafd93758d3ba71cc8a8642f72R36)